content/en/docs/instrumentation/js/serverless.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Serverless
+weight: 8
+---
+
+This guide will show you how to get started with tracing serverless functions using OpenTelemetry instrumentation libraries.
+
+## AWS Lambda
+
+Following the next steps will allow your AWS Lambda functions to use OpenTelemetry to send traces to the configured backend.
+
+### Dependencies
+
+Create an empty package.json:
+
+```sh
+npm init -y
+```
+
+Install dependencies used by the example.
+
+```sh
+npm install \
+  @opentelemetry/api \
+  @opentelemetry/auto-instrumentations-node \
+  @opentelemetry/exporter-trace-otlp-http \
+  @opentelemetry/instrumentation \
+  @opentelemetry/sdk-trace-base \
+  @opentelemetry/sdk-trace-node
+```
+
+### AWS Lambda wrapper code
+
+This file will contain all the OpenTelemetry logic, which will enable tracing.
+Please save the following code as `lambda-wrapper.js`.
+
+```javascript
+/* lambda-wrapper.js */
+
+const api = require("@opentelemetry/api");
+const { BatchSpanProcessor } = require("@opentelemetry/sdk-trace-base");
+const {
+  OTLPTraceExporter,
+} = require("@opentelemetry/exporter-trace-otlp-http");
+const { NodeTracerProvider } = require("@opentelemetry/sdk-trace-node");
+const { registerInstrumentations } = require("@opentelemetry/instrumentation");
+const {
+  getNodeAutoInstrumentations,
+} = require("@opentelemetry/auto-instrumentations-node");
+
+api.diag.setLogger(new api.DiagConsoleLogger(), api.DiagLogLevel.ALL);
+
+const provider = new NodeTracerProvider();
+const collectorOptions = {
+  url: "<backend_url>",
+};
+
+const spanProcessor = new BatchSpanProcessor(
+  new OTLPTraceExporter(collectorOptions)
+);
+
+provider.addSpanProcessor(spanProcessor);
+provider.register();
+
+registerInstrumentations({
+  instrumentations: [
+    getNodeAutoInstrumentations({
+      "@opentelemetry/instrumentation-aws-lambda": {
+        disableAwsContextPropagation: true,
+      },
+    }),
+  ],
+});
+```
+
+Replace `<backend_url>` with the URL of your favorite backend to export all
+traces to it. If you don't have one setup already, you can check out [Jaeger](https://www.jaegertracing.io/) or [Zipkin](https://zipkin.io/).
+
+Note that `disableAwsContextPropagation` is set to true. The reason for this is
+that the Lambda instrumentation tries to use the X-Ray context headers by
+default, this results in a non-sampled context, which creates a
+`NonRecordingSpan`.
```

Comment:
I believe this is only true if active tracing is disabled on your function.  When it is enabled then a sampled trace context will be available.

Reply:
True, since by default active tracing is not enabled, I've set this option to true

Reply:
```suggestion
Note that `disableAwsContextPropagation` is set to true. The reason for this is
that the Lambda instrumentation tries to use the X-Ray context headers by
default, this results in a non-sampled context unless active tracing is enabled for this function, which creates a
`NonRecordingSpan`.
```

That should be made explicit in the text.  Readers may be confused and think it is not at all possible to get a recording span or to use the AWS context propagation carrier.

Reply:
You're right, it should be noted. I pushed an update!